src/components/SearchBox/onSelectedFactory.ts

```diff
@@ -39,32 +39,38 @@ export const onHighlightFactory = (setPreview) => (e, location) => {
   setPreview({ ...getSkeleton(location), noPreviewButton: true });
 };
 
-const fitBounds = (location, panelShown = false) => {
-  const [w, s, e, n] = location.bbox;
-  const bbox = new maplibregl.LngLatBounds([w, s], [e, n]);
-  const panelWidth = panelShown ? 410 : 0;
-  getGlobalMap()?.fitBounds(bbox, {
-    padding: { top: 5, bottom: 5, right: 5, left: panelWidth + 5 },
-  });
+const fitBounds = (option, setView, panelShown = false) => {
+  if (!option.properties.extent) {
+    // TODO smoothly set view to [lat,lon] without known bbox
```

Comment:
TODO: smoothly set view to [lat,lon] without known bbox

Reply:
We could probably use `getGlobalMap()?.easeTo()`, but i havent found any issue? Can you please show me when its broken?

see https://maplibre.org/maplibre-gl-js-docs/api/map/#map#easeto

Reply:
Thanks for the documentation (maplibre.org). I've decided to use [`flyTo()`](https://maplibre.org/maplibre-gl-js-docs/api/map/#map#flyto). This part of code was mine, so I was only asking how to implenent this transition.